lib/datadog/kit/identity.rb

```diff
@@ -0,0 +1,36 @@
+# typed: false
+
+module Datadog
+  module Kit
+    # Tracking identity via traces
+    module Identity
+      # Attach user information to the trace
+      #
+      # Values must be strings. Keys are free form, although the following keys have specific meanings:
+      # - id: Mandatory. Username or client id extracted from the access token or
+      #   Authorization header in the inbound request from outside the system.
+      # - email: Email of the authenticated user associated to the trace.
+      # - name: User-friendly name. To be displayed in the UI if set.
+      # - session_id: Session ID of the authenticated user.
+      # - role: Actual/assumed role the client is making the request under
+      #   extracted from token or application security context.
+      # - scope: Scopes or granted authorities the client currently possesses
+      #   extracted from token or application security context. The value would
+      #   come from the scope associated with an OAuth 2.0 Access Token or an
+      #   attribute value in a SAML 2.0 Assertion.
```

Comment:
Regardless if we make changes to `data = {}`, could you reformat the documented keys in a YARD @param format?

Reply:
Sure!